e2e/exec.go

```diff
@@ -68,20 +56,19 @@ func execScriptOnVm(ctx context.Context, s *Scenario, vmPrivateIP, jumpboxPodNam
 	identifier := uuid.New().String()
 	var scriptFileName, remoteScriptFileName, interpreter string
 
-	switch script.interpreter {
-	case Powershell:
+	if s.IsWindows() {
```

Comment:
I think I prefer this as a switch statement because we might add different interpreters in future. But it's not a strong preference, so no need to change.